.github/workflows/rust.yml

```diff
@@ -37,5 +37,5 @@ jobs:
         run: RUSTFLAGS="-D warnings" rustc --version
       - name: Build
         run: cargo build --verbose
-      - name: Run tests
-        run: RUSTFLAGS="-D warnings" cargo test --verbose -- --test-threads=1
+#      - name: Run tests
+#        run: RUSTFLAGS="-D warnings" cargo test --verbose -- --test-threads=1
```

Comment:
We need this no?

Reply:
No, i moved it to `test.yml` file in case the tests grow up afterwards for more configuration.